backend/src/api/index.ts

```diff
@@ -11,6 +12,9 @@ const createApiRouter = (opts: Opts) => {
   router.use('/form', createFormRouter(opts));
   router.use('/keycloak', createKeycloakRouter(opts));
 
+  // add error handling middleware
+  router.use(errorMiddleware);
```

Comment:
do we need both occurrences of this? same line exists in backend/src/api/form.ts

Reply:
Nice catch! , fixed in [9554be9](https://github.com/formcapture/form-backend/pull/164/commits/9554be9a47270464f06feab662dd22dc3c8ff59e) by remove middleware usage in `api/form.ts`